docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -32,15 +34,15 @@ This file contains instructions for the exercise. It should explicitly explain w
 
 ## Step 3: add .docs/hints.md
 
-If the user gets stuck, we will allow them to click a button requesting a hint, which shows this file. We will softly discourage them using it. The file should contain both general and task-specific "hints". These hints should be enough to unblock almost any
+If the user gets stuck, we will allow them to click a button requesting a hint, which shows this file. We will softly discourage them using it. The file should contain both general and task-specific "hints". These hints should be enough to unblock almost any student.
 
 ## Step 4: add .docs/after.md (optional):
 
 Once the user completes the exercise they will be shown this file, which gives them any bonus information or further reading about the concept taught.
 
 These files are also all described in the [concept exercises document][docs-concept-exercises].
 
-## Step 5: update languages/<TRACK>/config.json
+## Step 5: update languages/&lt;TRACK&gt;/config.json
```

Comment:
Does `\< and \>` work for this as well, or does that also mess with rendering?

Reply:
I don't know actually!